### ISSUE TITLE
Can you please add a license?

### ISSUE THREAD

**Issue by mibzman** [unawaz/stochastic-hill-climbing#1]
Hello,
Could you please add an explicit 'LICENSE' file to the repo so that it's clear under what terms the content is provided, and under what terms contributions are licensed?

According to Github [here](https://choosealicense.com/no-permission/), this repo currently cannot be used for any purpose without your written permission.

**Comment by unawaz** [comment]
hi,
yes, that is my intention, i have adsense on there and want to earn some
money by my content.

The data comes from the o*net website, which is a government funded website
and the data comes from the US department of labour. I wrote this on the
page too. So you are free to get the data from there, it's only 1 file in
think, or somethi,g I downloaded it a few years ago s i don't remember
exactly

Sincerely
Umar Nawaz
